### DISCUSSION TITLE
mi practica pokedex

### DISCUSSION THREAD

**Discussion by LuisEsquivel27** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#444] [Q&A] [answered]
Hola, les comparto mi práctica por si tienen comentarios para mejora y jugar un rato para adivinar su pokemon jaja.

https://pokedexlaunch.web.app/ 

Tengo una duda, al cargar la aplicación en movil se muestra así 
![image](https://user-images.githubusercontent.com/26752004/158885672-a27e928e-75eb-4720-a3ce-881368df24e1.png)

Pero al buscar el pokemon se ajusta correctamente 
![image](https://user-images.githubusercontent.com/26752004/158885721-6c93829e-4efe-4244-adda-17e93dc04866.png)

Saben que podría estar afectando?

**Comment by micheldavrdz** [comment, marked as answer]
A lo que puedo ver tu section tiene un height asignado de 100vh, entonces toma 100% del tamaño del viewport aunque no haya nada para rellenarlo (entonces la "altura" de lo que ves en pantalla al inicio es: la altura de tu nav + 100% del viewport del section) por lo cual tienes que dar scroll para ver el resto

**Reply by LuisEsquivel27** [reply]
Si tienes razón, lo voy a modificar, gracias.

**Comment by Ricardo-g-garcia** [comment]
Esta padre el juego, muy bien
